packages/data-sql/src/converter/index.test.ts

```diff
@@ -45,6 +45,77 @@ test('Get all selects', () => {
 			},
 		],
 		from: sample.query.collection,
+		parameters: [],
+	};
+
+	expect(res).toStrictEqual(expected);
+});
+
+test('Get selects with a limit', () => {
+	sample.query.modifiers = {
+		limit: {
+			type: 'limit',
+			value: randomInteger(1, 100),
+		},
+	};
+
+	const res = convertAbstractQueryToAbstractSqlQuery(sample.query);
+
+	const expected: AbstractSqlQuery = {
+		select: [
+			{
+				type: 'primitive',
+				table: sample.query.collection,
+				column: (sample.query.nodes[0] as AbstractQueryFieldNodePrimitive).field,
+			},
+			{
+				type: 'primitive',
+				table: sample.query.collection,
+				column: (sample.query.nodes[1] as AbstractQueryFieldNodePrimitive).field,
+			},
+		],
+		from: sample.query.collection,
+		// @ts-ignore
+		limit: { parameterIndex: 0 },
+		// @ts-ignore
+		parameters: [sample.query.modifiers.limit.value],
+	};
+
+	expect(res).toStrictEqual(expected);
+});
+
+test('Get selects with a limit and offset', () => {
+	sample.query.modifiers = {
+		limit: {
+			type: 'limit',
+			value: randomInteger(1, 100),
+		},
+		offset: {
+			type: 'offset',
+			value: randomInteger(1, 100),
+		},
+	};
+
+	const res = convertAbstractQueryToAbstractSqlQuery(sample.query);
+
+	const expected: AbstractSqlQuery = {
+		select: [
+			{
+				type: 'primitive',
+				table: sample.query.collection,
+				column: (sample.query.nodes[0] as AbstractQueryFieldNodePrimitive).field,
+			},
+			{
+				type: 'primitive',
+				table: sample.query.collection,
+				column: (sample.query.nodes[1] as AbstractQueryFieldNodePrimitive).field,
+			},
+		],
+		from: sample.query.collection,
+		limit: { parameterIndex: 0 },
+		offset: { parameterIndex: 1 },
+		// @ts-ignore
+		parameters: [sample.query.modifiers.limit.value, sample.query.modifiers.offset.value],
```

Comment:
Let's use non-null assertion instead of the `ts-ignore` directive:
```suggestion
		limit: { parameterIndex: 0 },
		parameters: [sample.query.modifiers.limit!.value],
	};

	expect(res).toStrictEqual(expected);
});

test('Get selects with a limit and offset', () => {
	sample.query.modifiers = {
		limit: {
			type: 'limit',
			value: randomInteger(1, 100),
		},
		offset: {
			type: 'offset',
			value: randomInteger(1, 100),
		},
	};

	const res = convertAbstractQueryToAbstractSqlQuery(sample.query);

	const expected: AbstractSqlQuery = {
		select: [
			{
				type: 'primitive',
				table: sample.query.collection,
				column: (sample.query.nodes[0] as AbstractQueryFieldNodePrimitive).field,
			},
			{
				type: 'primitive',
				table: sample.query.collection,
				column: (sample.query.nodes[1] as AbstractQueryFieldNodePrimitive).field,
			},
		],
		from: sample.query.collection,
		limit: { parameterIndex: 0 },
		offset: { parameterIndex: 1 },
		parameters: [sample.query.modifiers.limit!.value, sample.query.modifiers.offset!.value],
```